model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt

```diff
@@ -3,7 +3,7 @@ package com.testerum.model.text.parts.param_meta.field
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.testerum.model.text.parts.param_meta.TypeMeta
 
-data class FieldTypeMeta (
-        @JsonProperty("name") val name: String,
-        @JsonProperty("type") val type: TypeMeta
+data class TypeMetaFieldDescriptor(
```

Comment:
Should this be renamed on the frontend too? 
If you change the model name on backend make sure you update the frontend too.

Reply:
nice catch, thanks

fixed